src/Symfony/Component/Console/Attribute/Ask.php

```diff
@@ -78,13 +103,25 @@ public static function tryFrom(\ReflectionParameter|\ReflectionProperty $member,
             $question->setValidator($self->validator);
             $question->setMaxAttempts($self->maxAttempts);
 
-            if ($reflection->isBackedEnumType()) {
+            if ($self->normalizer) {
+                $question->setNormalizer($self->normalizer);
+            } elseif (is_subclass_of($type->getName(), \BackedEnum::class)) {
                 /** @var class-string<\BackedEnum> $backedType */
                 $backedType = $reflection->getType()->getName();
                 $question->setNormalizer(fn (string|int $value) => $backedType::tryFrom($value) ?? throw InvalidArgumentException::fromEnumValue($reflection->getName(), $value, array_map(fn (\BackedEnum $enum): string|int => $enum->value, $backedType::cases())));
             }
 
-            $value = $io->askQuestion($question);
+            if ('array' === $type->getName()) {
+                $value = [];
+                while ($v = $io->askQuestion($question)) {
+                    if (\PHP_EOL === $v || ($question->isTrimmable() && '' === $v = trim($v))) {
```

Comment:
for testing, like done in #61501?
```suggestion
                    if ("\x4" === $v || \PHP_EOL === $v || ($question->isTrimmable() && '' === $v = trim($v))) {
```

Reply:
reading #61501, in QuestionHelper we don't trim to decide to break, isn't it?

Reply:
Added. thanks!

Reply:
>reading https://github.com/symfony/symfony/pull/61501, in QuestionHelper we don't trim to decide to break, isn't it?

From my tests, it behaves as expected in both cases, depending on whether `$question->isTrimmable()` is true.